### PR TITLE
fix test issues for 0.16.4

### PR DESCRIPTION
**Pull request by Kucharssim**
fixes https://github.com/jasp-stats/jasp-test-release/issues/1997
fixes https://github.com/jasp-stats/jasp-test-release/issues/1998
fixes https://github.com/jasp-stats/jasp-issues/issues/1829

The second issue is relatively difficult to solve nicely because the error occurs inside of the sampling function of the `conting` package. It also does not seem there is an easy check on the data to throw an error before running the sampling. So unless we go inside of the package and fiddle with that, we can only improve the error message.

### CODE REVIEW

File: R/regressionloglinearbayesian.R [hunk 1]
```diff
@@ -41,11 +41,11 @@ RegressionLogLinearBayesian <- function(jaspResults, dataset = NULL, options, ..
     if(options$counts != "")
       counts <- options$counts
     if(length(options$modelTerms) > 0)
-      factors <- options$modelTerms
-    return(.readDataSetToEnd(columns.as.factor = factors, 
+      factors <- options$factors
```
Comment (Kucharssim) [inline R/regressionloglinearbayesian.R:44]: This change is unrelated to the fixes. I just noticed that the using the `modelTerms` we unnecessarily duplicate columns in the data set if interactions are specified in the model.

**Review by vandenman — APPROVED** [review]
both issues are fixed!
